droidlet/memory/sql_memory.py

```diff
@@ -997,6 +1002,11 @@ def db_write(self, query: str, *args) -> int:
         if self.on_delete_callback is not None and deleted:
             self.on_delete_callback(deleted)
         self._db_write("DELETE FROM Updates")
+        # a sample filter for emitting to the dashboard
+        if "VoxelObjects" in query:
+            hook_data = {"name" : "db_write", "time" : datetime.now().strftime("%H:%M:%S"), "query" : query, "args" : args, "result" : r}
```

Comment:
Now the format of the data from db_write is query string, args list, etc. which is human-readable. You need a more structured format of data so that you can later use it to display on timeline (e.g. to show in a table, or support more fine-grained search). One way to do so is to convert the string to a key-value dict.